pkg/client/daemon/outbound.go

```diff
@@ -37,7 +37,7 @@ type outbound struct {
 
 	// Namespaces, accessible using <service-name>.<namespace-name>
 	namespaces map[string]struct{}
-	domains    map[string]iputil.IPs
+	domains    map[string]struct{}
```

Comment:
The `domains` map was actually a leftover from 2.2 but is now needed by the MacOS resolver for a slightly different purpose.